app/src/routes/api/v2/area.router.js

```diff
@@ -184,7 +183,7 @@ class AreaRouterV2 {
             // if it has subscription get subscription also and merge props
             // if it doesn't have subscription just return the area
         } else if (area.subscriptionId) {
-            area = await SubscriptionService.mergeSubscriptionSpecificProps(area, ctx.request.headers['x-api-key']);
+            [area] = await SubscriptionService.mergeSubscriptionSpecificProps([area], ctx.request.headers['x-api-key']);
```

Comment:
Can this block be skipped since it's called again in ln 212?

Reply:
removed in f7927c3